docs/operators/from_azure_blob_storage.md

```diff
@@ -0,0 +1,134 @@
+
+The characters `*` and `**` have a special meaning. `*` matches everything
+except `/`. `**` matches everything including `/`. The sequence `/**/` can also
+match nothing. For example, `container/**/data` matches `container/data`.
+
+Supported URI formats:
+
+1. `abfs[s]://[:<password>@]<account>.blob.core.windows.net[/<container>[/<path>]]`
+2. `abfs[s]://<container>[:<password>]@<account>.dfs.core.windows.net[/path]`
+3. `abfs[s]://[<account[:<password>]@]<host[.domain]>[<:port>][/<container>[/path]]`
+4. `abfs[s]://[<account[:<password>]@]<container>[/path]`
+
+(1) and (2) are compatible with the Azure Data Lake Storage Gen2 URIs, (3) is
+for Azure Blob Storage compatible service including Azurite, and (4) is a shorter
+version of (1) and (2).
+
+:::tip[Authenticate with the Azure CLI]
+Run `az login` on the command-line to authenticate the current user with Azure's
+command-line arguments.
+:::
+
+### `account_key = string (optional)`
+
+Account key for authenticating with Azure Blob Storage.
+
+### `watch = bool (optional)`
+
+In addition to processing all existing files, this option keeps the operator
+running, watching for new files that also match the given URL. Currently, this
+scans the filesystem up to every 10s.
+
+Defaults to `false`.
+
+### `remove = bool (optional)`
+
+Deletes files after they have been read completely.
+
+Defaults to `false`.
+
+### `rename = string -> string (optional)`
+
+Renames files after they have been read completely. The lambda function receives
+the original path as an argument and must return the new path.
+
+If the target path already exists, the operator will overwrite the file.
+
+The operator automatically creates any intermediate directories required for the
+target path. If the target path ends with a trailing slash (`/`), the original
+filename will be automatically appended to create the final path.
+
+### `path_field = field (optional)`
+
+This makes the operator insert the path to the file where an event originated
+from before emitting it.
+
+By default, paths will not be inserted into the outgoing events.
+
+### `{ … } (optional)`
+
+Pipeline to use for parsing the file. By default, this pipeline is derived from
+the path of the file, and will not only handle parsing but also decompression if
+applicable. This is using the same logic as [`from`](/reference/operators/from).
+
+## Examples
+
+### Read every JSON file from a container
+
+```tql
+from_azure_blob_storage "abfs://my-container/data/**.json"
+```
+
+### Read CSV files using account key authentication
+
+```tql
+from_azure_blob_storage "abfs://container/data.csv", account_key="your-account-key"
+```
+
+### Read Suricata EVE JSON logs continuously
+
+```tql
+from_azure_blob_storage "abfs://logs/suricata/**.json", watch=true {
+  read_suricata
+}
+```
+
+### Process files and move them to an archive container
+
+```tql
+from_azure_blob_storage "abfs://input/**.json", 
+  rename=path => "abfs://archive/" + path
```

Comment:
Does the prefixing with `abfs://` actually work? Even if it does, `path => "/archive/" + path` would be more idiomatic (although I'm not sure whether `path` starts with a slash or not).